appveyor.yml

```diff
@@ -8,12 +8,9 @@ services:
 
 shallow_clone: true
 
-cache:
```

Comment:
this was causing the problem? 🤔

Reply:
Appveyor was loading `tedious@2.3.1` for which I think everything was working properly as same tests will fail locally with `2.6.1`.

So I have removed caching entirely, as this is single test suite it should easily finish before its travis counterpart.